test/role.test.js

```diff
@@ -186,10 +195,16 @@ describe('role model', function() {
     });
 
     User.create({ name: 'Raymond', email: 'x@y.com', password: 'foobar' }, function(err, user) {
+      Role.isInRole('returnPromise', { principalType: ACL.USER, principalId: user.id },
+      function(yes) {
```

Comment:
Note that `Role.isInRole` is keeps callback in Node's error-first style. This should be `function(err, yes)` with the same body as in other tests below.